### ISSUE TITLE
2 assets not showing up on asset page

### ISSUE THREAD

**Issue by mbakeryo** [mlibrary/heliotrope#1699]
Area A of the Gabii Project's Excavations .jpg (Figure3_more.jpg) and New entrance to the courtyard .jpg (Figure4_more.jpg) are not showing up on their asset pages.
![screen shot 2018-03-30 at 7 46 53 am](https://user-images.githubusercontent.com/17436011/38136823-f798e154-33ee-11e8-8f18-90eb91f9218f.png)

**Comment by conorom** [comment]
This stuff is so tedious. They are the same on preview. I thought we discussed some automatic detection of these in the past. Could we add a post-characterization hook or something that flagged the job as failed if the height/width aren't set on relevant file types?

**Comment by sethaj** [comment]
Figure3_more.jpg: 4063x5675
Figure4_more.jpg: 2848x4288

Yeah, we probably should. We've already taken the characterization job from hyrax and put it in heliotrope so hooking into it should be possible. Seems like it could work. I'll make a ticket for it.

**Comment by conorom** [comment]
I ran the console command, just to be sure there are no others:
`FileSet.all.to_a.each { |f| puts "\"#{f.id}\",\"#{f.title.first.gsub('"','""')}\""  if (f.image? && (f.width.blank? || f.height.blank?)) } ;0`

There are no others on production. Preview is loaded with them (56 right now). I did notice that before, just wanted to point it out. Presumably something was wrong with the system config (FITS or whatever) when everything had to be re-imported last year. Just something I wanted to mention.
